### DISCUSSION TITLE
ERR_SSL_PROTOCOL_ERROR after installation

### DISCUSSION THREAD

**Discussion by joshuacadman01** [postalserver/postal#2135] [Help with using Postal]
I have followed the documentation exactly to install Postal on an Ubuntu server through Katapult. During installation, I received no error messages and assumed everything was fine, and even installed caddy. However, I cannot seem to get access to the UI when I type the hostname (that is forwarded to the IP and has ports open). 

When visiting post.DOMAIN.COM on Chrome, I receive the following error:

This site can’t provide a secure connection
post.DOMAIN.COM sent an invalid response.
ERR_SSL_PROTOCOL_ERROR

Has anyone had this issue before/or knows how to fix this?

**Comment by willpower232** [comment]
Based on 2133, looks like you figured it out in the end, this error usually means the certificate was not set up quite right.
